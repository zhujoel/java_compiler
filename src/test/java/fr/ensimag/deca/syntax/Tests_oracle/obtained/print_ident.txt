`> Program
   +> ListDeclClass [List with 0 elements]
   `> Main
      +> ListDeclVar [List with 2 elements]
      |  []> DeclVar
      |  ||  +> Identifier (int)
      |  ||  +> Identifier (a)
      |  ||  `> Initialization
      |  ||     `> Int (5)
      |  []> DeclVar
      |      +> Identifier (int)
      |      +> Identifier (b)
      |      `> Initialization
      |         `> Int (6)
      `> ListInst [List with 2 elements]
         []> Print
         ||  `> ListExpr [List with 2 elements]
         ||     []> Identifier (a)
         ||     []> Identifier (b)
         []> Println
             `> ListExpr [List with 2 elements]
                []> Identifier (a)
                []> Identifier (b)
