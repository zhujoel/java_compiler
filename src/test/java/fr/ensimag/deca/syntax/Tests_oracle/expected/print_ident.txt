`> Program
   +> ListDeclClass [List with 0 elements]
   `> Main
      +> ListDeclVar [List with 1 elements]
      |  []> DeclVar
      |      +> Identifier (int)
      |      +> Identifier (a)
      |      `> Initialization
      |         `> Int (5)
      `> ListInst [List with 1 elements]
         []> Print
             `> ListExpr [List with 1 elements]
                []> Identifier (a)
